vendor/engines/secure_rooms/app/models/secure_rooms/occupancy.rb

```diff
@@ -2,6 +2,9 @@ module SecureRooms
 
   class Occupancy < ActiveRecord::Base
 
+    alias_attribute :actual_start_at, :entry_at
```

Comment:
This will conflict with #1023 because I have it there as well. Just need to remember to ensure we don't have two sets of them.